src/label_map.cpp

```diff
@@ -190,16 +190,18 @@ int LabelMap::find_or_create(std::string mylabel,
   }
 
   // if label cannot be found or created, need more space reserved
-
```

Comment:
LAMMPS convention is to leave a space after comments

Reply:
fixed it